udata/core/reports/api.py

```diff
@@ -0,0 +1,34 @@
+from flask import request
+from flask_login import current_user
+import mongoengine
+
+from udata.api import api, API
+from udata.api_fields import patch
+from .models import Report
+
+ns = api.namespace('reports', 'User reported objects related operations (beta)')
+
+@ns.route('/', endpoint='reports')
+class ReportsAPI(API):
+    @api.doc('list_reports')
+    @api.expect(Report.__index_parser__)
+    @api.marshal_with(Report.__page_fields__)
+    def get(self):
+        query = Report.objects.visible()
```

Comment:
I'm getting an error `AttributeError: 'UDataQuerySet' object has no attribute 'visible'` on this line.

But with only `.objects` it's working great !

Reply:
Oups… Done in 7c2556828